packages/react-storage/src/components/StorageManager/types.ts

```diff
@@ -120,9 +120,12 @@ export interface StorageManagerProps {
    */
   showThumbnails?: boolean;
   /**
-   * A path to put files in the s3 bucket.
-   * This will be prepended to the key sent to
-   * s3 for each file.
```

Comment:
Don't think we should remove the description of the prop since it is still usable

Reply:
Yeah, I may delete it accidentally